build.sbt

```
@@ -68,7 +106,6 @@ val commonSettings: immutable.Seq[Def.Setting[_]] = List(
     "-deprecation",
     "-encoding",
     "UTF-8",
```

Getting rid of `-target:jvm-1.8` is good - we can also add [`-release:11`](https://docs.scala-lang.org/overviews/compiler-options/#:~:text=specific%20features%20removed.-,%2Drelease%20RELEASE,-or%20%2D%2Drelease%20RELEASE) in, as the best way of enforcing that a project is compiled with Java 11 or above:

```suggestion
    "UTF-8",
    "-release:11"
```